README.md

```diff
@@ -1,2 +1,67 @@
 # PyTOM GPU template matching for cryo-ET
 PyTOM's GPU template matching as a single pip plugin that can only be run from the command line.
+
+The full PyTOM repository can be found at: https://github.com/SBC-Utrecht/PyTom
+
+## Citation
+
+...
+
+## Requires
+
+```
+miniconda3
+nvidia-cuda-toolkit
```

Comment:
Currently the code needs cupy, so its not optional. I would like to add some numpy implementation as well which is not a large task, see #14. After it will be optional!